https://gitlab.freedesktop.org/freetype/freetype/-/commit/e82371bf6ec2efdf79f9ed2d00e633ba0bbf7830

From e82371bf6ec2efdf79f9ed2d00e633ba0bbf7830 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sun, 6 Nov 2022 05:18:59 +0000
Subject: [PATCH] Fix `-Wstrict-prototypes`.

* builds/unix/configure.raw: Fix `-Wstrict-prototypes`.
Clang 16 warns on these and they will be dropped in C23.

* builds/unix/freetype2.m4: Ditto.

Signed-off-by: Sam James <sam@gentoo.org>
--- a/builds/unix/configure.raw
+++ b/builds/unix/configure.raw
@@ -50,7 +50,7 @@ if test ${cross_compiling} = yes; then
 
   AC_MSG_CHECKING([for suffix of native executables])
   rm -f a.* b.* a_out.exe conftest.*
-  echo > conftest.c "int main() { return 0;}"
+  echo > conftest.c "int main(void) { return 0;}"
   ${CC_BUILD} conftest.c || AC_MSG_ERROR([native C compiler is not working])
   rm -f conftest.c
   if test -x a.out -o -x b.out -o -x conftest; then
--- a/builds/unix/freetype2.m4
+++ b/builds/unix/freetype2.m4
@@ -121,7 +121,7 @@ AC_DEFUN([AC_CHECK_FT2],
 #include <stdlib.h>
 
 int
-main()
+main(void)
 {
   FT_Library library;
   FT_Error  error;
GitLab
